examples/chef/common/clusters/keypad-input/KeypadInputManager.h

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/keypad-input-server/keypad-input-server.h>
+
+using chip::app::CommandResponseHelper;
+using KeypadInputDelegate = chip::app::Clusters::KeypadInput::Delegate;
+using SendKeyResponseType = chip::app::Clusters::KeypadInput::Commands::SendKeyResponse::Type;
+using CecKeyCodeType      = chip::app::Clusters::KeypadInput::CECKeyCodeEnum;
```

Comment:
no using in headers.
Comments from other headers apply: why is the featuremap a member with a TODO on it?

Reply:
Removed using from top scope. (Just leave it in class scope)
Remove the featuremap which is use less